packages/platform/atoms/availability/wrappers/PlatformAvailabilitySettingsWrapper.tsx

```diff
@@ -33,10 +35,10 @@ export const PlatformAvailabilitySettingsWrapper = ({
   onUpdateError,
   onUpdateSuccess,
 }: PlatformAvailabilitySettingsWrapperProps) => {
-  const { isLoading, data: schedule } = useClientSchedule(id);
-  const mySchedule = schedule as ApiSuccessResponse<ScheduleWithAvailabilitiesForWeb>;
+  const { isLoading, data: schedule } = useSchedule(id);
+  const { data: schedules } = useSchedules();
```

Comment:
this seems like a waste to fetch twice schedules, we could just fetch from useSchedules and extract the right schedule from it based on it